internal/extsvc/gerrit/client.go

```diff
@@ -102,13 +99,22 @@ func (c *Client) GetGroup(ctx context.Context, groupName string) (Group, error)
 	return respGetGroup, nil
 }
 
-// ListProjectsArgs defines options to be set on ListProjects method calls.
-type ListProjectsArgs struct {
-	Cursor *Pagination
-}
+func (c *Client) ListGroupMembers(ctx context.Context, groupID string) (ListAccountsResponse, error) {
 
-// ListProjectsResponse defines a response struct returned from ListProjects method calls.
-type ListProjectsResponse map[string]*Project
+	urlGroup := url.URL{Path: fmt.Sprintf("a/groups/%s/members/", groupID)}
+
+	reqAllAccounts, err := http.NewRequest("GET", urlGroup.String(), nil)
+
+	if err != nil {
+		return ListAccountsResponse{}, err
+	}
+
+	respListGroupMembers := ListAccountsResponse{}
+	if _, err = c.do(ctx, reqAllAccounts, &respListGroupMembers); err != nil {
+		return respListGroupMembers, err
```

Comment:
```suggestion
		return nil, err
```
to be more consistent with existing code